apps/document-storage/src/server/swagger.js

```diff
@@ -14,24 +14,33 @@ const document = {
 	tags: [
 		{
 			name: '',
-			description: '',
-		},
+			description: ''
+		}
 	],
 	securityDefinitions: {},
 	definitions: {
-        Documents: [
-            {
-                name: 'filename.pdf',
-                metadata: {
-                    documentType: 'application'
-                }
-            }
-        ]
-    },
+		createBlobUrl: [
+			{
+				caseType: 'application',
+				caseReference: '1',
+				documentName: 'PINS1',
+				GUID: 'D987654321',
+				blobStoreUrl: '/application/1/D987654321/PINS1'
+			}
+		],
+		Documents: [
+			{
+				name: 'filename.pdf',
+				metadata: {
+					documentType: 'application'
+				}
+			}
+		]
+	},
 	components: {}
 };
 
 const outputFile = './src/server/swagger-output.json';
-const endpointsFiles = ['./src/server/app.js'];
+const endpointsFiles = ['./src/server/app/routes.js'];
```

Comment:
Remember we spoke about doing this using the `*` wildcards? Could you please add that here?

Reply:
yes, I think we tried several versions, and this was the one that worked in the end, right?

Reply:
perhaps, however currently - `const endpointsFiles = ['./src/server/app/*.routes.js'] `is not returning anything, and neither is ` const endpointsFiles = ['./src/server/app/**/*.routes.js'] `
do you have any other suggestions?